pkg/dashboard/dashboard.go

```diff
@@ -15,19 +15,33 @@
 package dashboard
 
 import (
+	"math"
 	"net/http"
+	"strconv"
 
 	"github.com/gorilla/mux"
 	"k8s.io/klog/v2"
 
 	"github.com/fairwindsops/goldilocks/pkg/summary"
 )
 
+const kibabyte = 1024
+const mebibyte = kibabyte * 1024
+const gibibyte = mebibyte * 1024
+const kilobyte = 1000
+const megabyte = kilobyte * 1000
```

Comment:
Nit: It's a bit cleaner to use this syntax (also include the other const down below. 

```suggestion
const (
    kibabyte = 1024
    mebibyte = kibabyte * 1024
    gibibyte = mebibyte * 1024
    kilobyte = 1000
    megabyte = kilobyte * 1000
)
```

Reply:
I updated the code. Thank you!